src/vmm/src/vstate.rs

```diff
@@ -563,6 +540,34 @@ impl Vm {
             .map_err(Error::VmSetIrqChip)?;
         Ok(())
     }
+
+    pub(crate) fn set_kvm_memory_regions(
+        &self,
+        guest_mem: &GuestMemoryMmap,
+        track_dirty_pages: bool,
+    ) -> Result<()> {
+        let flags = if track_dirty_pages {
+            KVM_MEM_LOG_DIRTY_PAGES
+        } else {
+            0
+        };
+        guest_mem
+            .with_regions(|index, region| {
+                let memory_region = kvm_userspace_memory_region {
+                    slot: index as u32,
+                    // It's safe to unwrap because the guest address is valid.
```

Comment:
Is this comment accurate?

Reply:
It belongs 2 lines below, where https://github.com/firecracker-microvm/firecracker/pull/1693#discussion_r423624542 noticed it missing.